jihoo/GoogleLoginProject/src/main/java/com/gdg/googleloginproject/config/SecurityConfig.java

```diff
@@ -0,0 +1,45 @@
+package com.gdg.googleloginproject.config;
+
+import com.gdg.googleloginproject.jwt.JwtFilter;
+import com.gdg.googleloginproject.jwt.TokenProvider;
+import lombok.RequiredArgsConstructor;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+import org.springframework.security.config.annotation.web.builders.HttpSecurity;
+import org.springframework.security.config.annotation.web.configurers.AbstractHttpConfigurer;
+import org.springframework.security.config.http.SessionCreationPolicy;
+import org.springframework.security.crypto.factory.PasswordEncoderFactories;
+import org.springframework.security.crypto.password.PasswordEncoder;
+import org.springframework.security.web.SecurityFilterChain;
+import org.springframework.security.web.authentication.UsernamePasswordAuthenticationFilter;
+
+@Configuration
+@RequiredArgsConstructor
+public class SecurityConfig {
+
+    private final TokenProvider tokenprovider;
+
+    @Bean
+    public SecurityFilterChain filterChain(HttpSecurity httpSecurity) throws Exception {
+        return httpSecurity
+                .httpBasic(AbstractHttpConfigurer::disable)
+                .csrf(AbstractHttpConfigurer::disable)
+                .sessionManagement(sessionManagement -> sessionManagement.sessionCreationPolicy(
+                        SessionCreationPolicy.STATELESS))
+                .formLogin(AbstractHttpConfigurer::disable)
+                .logout(AbstractHttpConfigurer::disable)
+                .authorizeHttpRequests(authorizeRequests -> authorizeRequests
+                        .requestMatchers("/api/**").permitAll()
+                        .requestMatchers( "recipe/**").authenticated()
+                        .requestMatchers("/user", "/user/login").permitAll()
+                        .anyRequest().authenticated()

```

Comment:
```suggestion
                .authorizeHttpRequests(authorizeRequests -> authorizeRequests
                        .requestMatchers("/api/**").permitAll()
                        .requestMatchers( "/recipe/**").authenticated() // 이 부분
                        .requestMatchers("/user", "/user/login").permitAll()
                        .anyRequest().authenticated()

```

단순 오타인 것 같지만, `/`를 추가하거나 모두 빼서 일관성을 지키면 더 좋을 것 같아요!

Reply:
앗 다시 리펙토링 하겠습니다 !!